scripts/i18next/generate.ts

```diff
@@ -0,0 +1,280 @@
+import { existsSync, mkdirSync, readdirSync, readFileSync, realpathSync, writeFileSync } from 'fs';
+import { dirname, join, relative } from 'path';
+import { execSync } from 'child_process';
+
+import prettier from 'prettier';
+
+const NODE_MODULES_DIRECTORY = join(process.cwd(), 'node_modules');
+const PLATFORMS_DIRECTORY = join(process.cwd(), 'platforms');
+
+type SimplePackageJSON = {
+  dependencies?: Record<string, string>;
+  devDependencies?: Record<string, string>;
+  peerDependencies?: Record<string, string>;
+};
+
+type Config = {
+  /**
+   * The path of the locales folder
+   */
+  path: string;
+  /**
+   * The translation output path with placeholders for locale and namespace
+   */
+  output: string;
+  /**
+   * Bundle the translations with the build output (resources)
+   */
+  bundle: boolean;
+  /**
+   * The path where the resources are bundled
+   */
+  bundlePath: string;
+  /**
+   * The path where the resources file is created
+   */
+  resourcesPath: string;
+};
+
+function getWorkspacePackages(packageJSON: SimplePackageJSON) {
+  const deps = Object.keys(packageJSON.dependencies);
+  const devDeps = Object.keys(packageJSON.devDependencies);
+  const peerDeps = Object.keys(packageJSON.peerDependencies);
+
+  const packages = [...deps, ...devDeps, ...peerDeps].filter((name) => name.startsWith('@jwp/'));
```

Comment:
Would it be safer to use `yarn workspaces info` to figure out which packages are local, so we don't have to rely on the `@jwp` prefix?

Reply:
Hmm interesting, running `yarn workspaces info` returns the following for the web platform:

```json
{
  "@jwp/ott-web": {
    "location": "platforms/web",
    "workspaceDependencies": [
      "stylelint-config-jwp"
    ],
    "mismatchedWorkspaceDependencies": []
  }
```

The stylelint-config-jwp package is the only package installed in the `devDependencies`. The rest is still listed in the `peerDependencies`. I do remember we talked about this before. This indeed seems to suggest we should not be using `peerDependencies` for the workspace packages...